modules/lambda/main.tf

```diff
@@ -4,7 +4,7 @@ module "label" {
 }
 
 resource "aws_lambda_function" "autospotting" {
-  count = var.lambda_zipname != "" ? 0 : 1
+  count = var.lambda_zipname == "" ? 0 : 1
```

Comment:
This is also more clear than the negative ("if it is not null")

Reply:
Could you also change the line for S3-based binaries to use similar logic?

https://github.com/AutoSpotting/terraform-aws-autospotting/blob/33151a3639974c26967cae97b887da684a06447e/modules/lambda/main.tf#L39

Only difference is that it should have the opposite effect:
```hcl
  count = var.lambda_zipname == "" ? 1 : 0
```